controllers/lvmcluster_controller_watches.go

```diff
@@ -18,6 +18,7 @@ package controllers
 
 import (
 	"context"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
I think in some files github pkgs are ordered after k8s pkgs, but in general it would be `goimports` right? anyhow tried to adjust, maybe worth storing editor config for this some time